app/controllers/users_controller.rb

```diff
@@ -18,10 +26,47 @@ def create
     end
   end
 
+  def edit
+    @user = User.find_by id: params[:id]
+  end
+
+  def update
+    @user = User.find_by id: params[:id]
```

Comment:
What will happen if it don't find the user? How you can handle this situation(if @user is nil)